### PR TITLE
solution

### PR DESCRIPTION
**Pull request by MaxSchmide**
[DEMO LINK](https://maxschmide.github.io/react_people-table-advanced/)

### CODE REVIEW

**Review by GGsmou — APPROVED** [review]
GJ, approving!
You forgot to save params on opening people tab, but did it on selecting person, so approving)

File: src/pages/PeoplePage.tsx [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+        mother: getParent(result, person, 'mother'),
+      }));
+
+      setPeople(prepared);
+    })
+      .catch(() => setIsError(true))
+      .finally(() => setIsPending(false));
+  };
+
+  const visiblePeople = filterAndSortPeople(people,
```
Comment (anstsot) [inline src/pages/PeoplePage.tsx:51]: use `useMemo` to avoid recalculations on rerender

File: src/components/PeopleFilters.tsx [hunk 2]
```diff
@@ -16,6 +67,8 @@ export const PeopleFilters = () => {
             type="search"
             className="input"
             placeholder="Search"
+            value={query}
+            onChange={e => handleSearchTerms(e)}
```
Comment (anstsot) [inline src/components/PeopleFilters.tsx:71]:
```suggestion
            onChange={handleSearchTerms}
```

File: src/components/PeopleFilters.tsx [hunk 3]
```diff
@@ -27,66 +80,78 @@ export const PeopleFilters = () => {
...
-              className="button mr-1"
-              href="#/people?centuries=20"
+              className={classNames('button mr-1', {
+                'is-info': centuries.includes('20'),
+              })}
+              params={{ centuries: handleCenturiesParams('20') }}
             >
               20
-            </a>
+            </SearchLink>
```
Comment (anstsot) [inline src/components/PeopleFilters.tsx:131]: create array of values

**Review by anstsot — COMMENTED** [review]
Let’s add a few fixes to improve your code and fix bug:
- The search should include mother name and father name as well
image

**Review by anstsot — CHANGES_REQUESTED** [review]
Let’s add a few fixes to improve your code and fix bug:
- The search should include mother name and father name as well
image

**Review by And678 — APPROVED** [review]
All fixed, good job :white_check_mark:

**Pull request closed without merging**